sections/clause_06_overview.adoc

```diff
@@ -50,14 +50,14 @@ For the Things whose location changed, the HistoricalLocations entities offer th
 A Thing also can have multiple Datastreams.
 A Datastream is a collection of Observations grouped by the same ObservedProperty, Sensor and optionally by Feature and ObservingProcedure.
 An Observation is an event performed by a Sensor that produces a result whose value is an estimate of an ObservedProperty of any given Feature which may be a proximate or ultimate FeatureofInterest.
-Details of each above described entity are provided in <<sensing-entities1>>.
+Details of each above described entity are provided in <<sensing-core>>.
 
 The Sampling part is a new addition to the SensorThings API.
 As is often the case, an Observation may not be a true representation of the intended Feature's Property that an Observer may be trying to Observe.
 Sampling is a common strategy to understand the characteristics of an otherwise difficult-to-measure Property of any feature-of-interest.
-In order to generate Samplings, a Sampler, that may be any physical device or even a human being part of a survey campaign, must carefully select a SamplingProcedure.
-Samplings may be generated by a sequence of SamplingProcedures (and vice-versa), however a Sampler must employ a unique SamplingProcedure to maintain unique Sampling-Sampler-SamplingProcedure relationships.
-In scenarios where a Feature is not directly available for Sampling, a PreparationProcedure composed of multiple PreparationSteps may optionally be used to generate a PreparedFeature.
+In order to generate Samplings, a Sampler, that may be any physical device (or even a human being part of a survey campaign), must carefully select a SamplingProcedure.
+A Sampling may be generated by a SamplingProcedure. This SamplingProcedure can be used by multiple Samplers, however a Sampler can not use multiple SamplingProcedures. Hence, any Sampling that is generated by a Sampler is always associated with a unique SamplingProcedure.
+In scenarios where a Feature is not directly available for Sampling, a PreparationProcedure composed of multiple PreparationSteps may optionally be used to generate a PreparedSample (Feature entity).
```

Comment:
> In scenarios where a Feature is not directly available for Sampling

What does that mean?
From OMS: In addition, various preparation steps may be performed on samples both before and after observations are performed on the sample.

Reply:
It means when a Feature is basically a prepared Feature (or Sample). For example, in case of water quality monitoring, it is not feasible to have the entire River Feature available to observe, a PreparationProcedure is applied to get a sample of water

Reply:
No, preparation procedures are not used to create a sample. Sampling procedures create samples. Preparation procedures are applied to samples to prepare them for measuring, or to conserve them like with specimens.